core/src/main/java/it/pagopa/selfcare/mscore/core/util/OnboardingInstitutionUtils.java

```diff
@@ -77,7 +78,7 @@ public static RelationshipState getStatus(InstitutionType institutionType, Onboa
             case PG:
                 return RelationshipState.ACTIVE;
             default:
-                if (InstitutionType.GSP == institutionType && request.getProductId().equals("prod-interop") && institution.getOrigin().equals("IPA")){
+                if (InstitutionType.GSP == institutionType && request.getProductId().equals("prod-interop") && institution.getOrigin().equals("IPA")) {
```

Comment:
could be better using constant or enum than hardcoded string, for products such as "prod-interop" 
you could introduce a enum